Description: Replace stray POD with =for comment
 This is the only documented methd and leaving in =head2 makes the build
 system generate a manual page lacking a basic requisite like the whatis
 entry (=head1 NAME)
Forwarded: http://mail-archives.apache.org/mod_mbox/perl-modperl/201004.mbox/thread
Author: Damyan Ivanov <dmn@debian.org>
Last-Update: 2011-03-22
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>
--- a/Apache-Test/lib/Apache/TestServer.pm
+++ b/Apache-Test/lib/Apache/TestServer.pm
@@ -379,7 +379,7 @@
     }
 }
 
-=head2 stop()
+=for comment head2 stop()
 
 attempt to stop the server.
 
